drivers/gl_context/SCsub

```diff
@@ -2,7 +2,8 @@ Import('env')
 Import('env_drivers')
 
 env_drivers.add_source_files(env.drivers_sources,"*.cpp")
-env_drivers.add_source_files(env.drivers_sources,"glew.c")
-
-env_drivers.Append(CFLAGS = ['-DGLEW_STATIC'])
-env_drivers.Append(CPPPATH = ['.'])
+if (env['glew'] == 'yes'):
```

Comment:
Apparently that won't work if `env['glew']` was not defined, so I'll have to use `if ('glew' in env and env['glew'] == 'yes'`.